x/ibc/02-client/client/rest/query.go

```diff
@@ -127,6 +128,35 @@ func queryRootHandlerFn(cliCtx context.CLIContext, queryRoute string) http.Handl
 	}
 }
 
+// queryCommitterHandlerFn implements a committer querying route
```

Comment:
Is there a way to automatically generate the `@` prefixed docs in this file or does it have to be done manually?

Reply:
they need to be added manually